### DISCUSSION TITLE
Rubrica Final del Proyecto Grupo4: Auto

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Electronica-Microcontrolada/Grupo4#54] [Announcements]
Nota de la 1er Semana: 8(Ocho)

Que tal estimados, trabajaron muy bien. 
a) Diagramaron el proyecto
b) Perfilaron el proyecto temporalmente con un diagrama gantt
c) Utilizaron correctamente la documentacion provista por el profesor Jorge Morales @JorEl057 
Ademas proyectaron una aplicacion sobre el proyecto general, y esto es sumamente bueno. Tener la habilidad de tomar un proceso o diseño existente y adaptarlo a una idea propia o provechosa e innovadora es una caracteristica de los alumnos que nosotros valoramos mucho. Los felicitamos, continuen asi. 
Pero, no realizaron la presentacion de su proyecto. Recuerden que las presentaciones audiovisuales son muy importantes para conseguir patrocinio. No descuiden esto, porque la presentacion de la idea puede llegar a ser mas importante, en instancias preliminares, que el desarrollo mismo.

**Comment by guelo2019** [comment]
Muchas gracias por la devolución!

**Comment by Gona79** [comment]
Nota de 3er Semana: 9(Nueve)

Chicos el objetivo del desarrollo del proyecto era conocer las herramientas de:
Creacion de proyectos ( Gantt, AGILE, Proyectos)
Ideacion, desarrollo y prueba.
Pero principalmente el trabajo colaborativo. Recuerden utilizar el kanban de github, no lo dejen abierto. Cierren las issues, cierren el proyecto. 
Lo hicieron muy bien, adapten este trabajo en forma de plantilla y les sera util para el resto de los proyectos de la cursada. Continuen asi.

**Reply by guelo2019** [reply]
Muchas  gracias  por la devolución!.
